plugin/clone/include/clone.h

```diff
@@ -41,6 +41,9 @@ Clone Plugin: Common objects and interfaces
 #include "mysql/components/services/log_builtins.h"
 #include "violite.h"
 
+#define DEFAULT(X) X
```

Comment:
These two are only used in the .cc file, so should be declared there. But for these particular macros, no need to have them at all I believe, i.e. the DEFAULT one already has a comment next to it at the use place.